Makefile

```diff
@@ -1,20 +1,20 @@
 .PHONY: build compile run
 
 build:
-	docker build -t fetcher .
+	docker build -t mozart_fetcher .
 
 compile:
 	docker run \
 		--rm=true \
 		-v $(CURDIR):/opt/app \
 		-w /opt/app \
 		-e "DEV_CERT_PEM=$(DEV_CERT_PEM)" \
-		-e "APP_NAME=fetcher" \
+		-e "APP_NAME=mozart_fetcher" \
 		-e "APP_VERSION=0.1.0" \
 		-e "MIX_ENV=prod" \
-		-e "RELEASE_DIR=_build/prod/rel/fetcher/releases/0.1.0/" \
+		-e "RELEASE_DIR=_build/prod/rel/mozart_fetcher/releases/0.1.0/" \
```

Comment:
Shame about all these hardcoding, such as version number. I know it's already in there and this is just a rename.